### DISCUSSION TITLE
http-c2.json not affecting implant generation

### DISCUSSION THREAD

**Discussion by dalemazza** [BishopFox/sliver#1363] [Q&A] [answered]
anyone here played with the http-c2.json config file? ive managed to get the server-config section to work with a cache header but for the life of me, it doesnt matter what i change any of the implant values to, the implant just carries on calling back to the default values?

I am using a server on an ubunutu box and the client on my Kali machine. I am also editing the ~/.sliver/configs/http-c2.json file like below, saving it then connecting with sliver to the server and generating a new implant.

![image](https://github.com/BishopFox/sliver/assets/66513243/2645169d-9e93-4c27-9072-abf5c6c16513)

But alas the implant still seems to generate with the default values.

**Comment by moloch--** [comment, marked as answer]
Double check it's the correct home directory, i.e. if the server is running as root so you'll need to put the config in `/root/.sliver/configs`

**Reply by dalemazza** [reply]
i will give this a go. I would have thought it would use my local config file on my client VM and not use the config on my Team Server, But yes yes the Team server is running as root

**Reply by moloch--** [reply]
It's a server-side config since the server must know how to parse the URLs.
